guides/tutorials/issue-a-credential.md

```diff
@@ -106,90 +53,57 @@ ngrok http <PORT>
-
-When we want to issue a credential we must first create a blueprint, the
-schema, of the credential and bind it to a specific issuer, the credential
-definition. This binding makes sure that when you want to verify the
-credential, you can request that it must be issued from a specific party.
-
-:::issuer
+#### Holder
 
-```typescript showLineNumbers issue-a-credential.ts section-3
+For the _Holder_ we need to setup a basic agent with a wallet, mediator, outbound transport and a ledger.
 
-```
+> If you want to follow this tutorial in a mobile environment:
+>
+> 1. Use the `agentDependencies` from `@aries-framework/react-native`
+> 1. It is very important to note that mobile agents do not support HTTP by default. It is recommended to do everything over HTTPS, but for development HTTP can be enabled for [iOS](https://stackoverflow.com/questions/30731785/how-do-i-load-an-http-url-with-app-transport-security-enabled-in-ios-9) and [Android](https://stackoverflow.com/questions/51902629/how-to-allow-all-network-connection-types-http-and-https-in-android-9-pie).
 
----
+:::holder
 
-```typescript showLineNumbers issue-a-credential.ts section-4
+```typescript showLineNumbers issue-a-credential.ts section-2
 
 ```
 
 :::
 
 ### 3. Listening for incoming credentials
 
-When we want to accept a credential, we have to listen to incoming credentials
-and handle accordingly. In this example we do not have any user interaction,
-but is likely that your application would have a user-interface which would
-display the credential. When receiving a credential offer you can get the
-values from `credentialExchangeRecord.credentialAttributes`.
+When we want to accept a credential, we have to listen to incoming credentials and handle accordingly. In this example we do not have any user interaction, but is likely that your application would have a user-interface which would display the credential. When receiving a credential offer you can get the values from `credentialExchangeRecord.credentialAttributes`.
 
 :::holder
 
-```typescript showLineNumbers issue-a-credential.ts section-5
+```typescript showLineNumbers issue-a-credential.ts section-3
 
 ```
 
 :::
 
 ### 4. Issuing a credential
 
-Now that everything is setup on both sides, the _Issuer_ can now offer a
-credential to the _Holder_.
+Now that everything is setup on both sides, the _Issuer_ can now offer a credential to the _Holder_.
 
-> In this example we do not instantiate a connection and assume that there is
-> one. Please refer to this guide [Create a connection](./create-a-connection)
-> to get a connection and connectionId
+> In this example we do not instantiate a connection and assume that there is one. Please refer to this guide [Create a connection](./create-a-connection) to get a connection and connectionId.
 
 :::issuer
 
-```typescript showLineNumbers issue-a-credential.ts section-6
+```typescript showLineNumbers issue-a-credential.ts section-4
```

Comment:
Should we add tabs here, that show how to do it with both indy and the new anoncreds format? The only thing different is that indy can only be used with unqualified identifiers, and that the key is `indy` instead of `anoncreds`